plugins/inventory/flightctl.py

```
@@ -0,0 +1,444 @@
+        # coerce types
+        if op in ('>', '>=', '<', '<='):
+            # attempt numeric or timestamp
+            try:
+                a = float(actual)
+                b = float(val)
+            except ValueError:
+                a = date_parser.isoparse(actual)
+                b = date_parser.isoparse(val)
+        if op in ('=', '==', '!=', 'in', 'notin', 'contains', 'notcontains'):
+            # for in/notin operators, val should eval() to list
+            if op in ('in', 'notin'):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)